seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpGetCommandProcessor.java

```diff
@@ -752,7 +752,7 @@ private JsonObject getJobInfoJson(JobState jobState, String jobMetrics, JobDAGIn
                         DateTimeUtils.toString(
                                 jobState.getFinishTime(),
                                 DateTimeUtils.Formatter.YYYY_MM_DD_HH_MM_SS))
-                .add(RestConstant.JOB_DAG, JsonUtils.toJsonString(jobDAGInfo))
+                .add(RestConstant.JOB_DAG, jobDAGInfo.toJsonObject())
```

Comment:
Seem like we need add test case to check job dag value?

Reply:
> Seem like we need add test case to check job dag value?

Done.